_topic_map.yml

```diff
@@ -1633,35 +1633,37 @@ Name: Logging
 Dir: logging
 Distros: openshift-enterprise,openshift-origin,openshift-dedicated
 Topics:
-- Name: About cluster logging
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
   File: cluster-logging
-- Name: Installing cluster logging
+- Name: Installing Logging
   File: cluster-logging-deploying
   Distros: openshift-enterprise,openshift-origin
-- Name: Installing the Cluster Logging and Elasticsearch Operators
+- Name: Installing the Logging and Elasticsearch Operators
   File: dedicated-cluster-deploying
   Distros: openshift-dedicated
-- Name: Configuring your cluster logging deployment
+- Name: Configuring your Logging deployment
   Dir: config
   Distros: openshift-enterprise,openshift-origin
   Topics:
-  - Name: About the Cluster Logging Custom Resource
+  - Name: About the Logging Custom Resource
```

Comment:
s/Custom Resource/custom resource (my error not to have updated this)

Reply:
Updated.